ad-insertion/video-analytics-service/build.sh

```diff
@@ -3,4 +3,35 @@ DIR=$(dirname $(readlink -f "$0"))
 
 cp -f "$DIR/../../script/messaging.py" "$DIR/feeder"
 cp -f "$DIR/../../script/zkstate.py" "$DIR/feeder"
-. "$DIR/../../script/build.sh"
+
+USER="docker"
+GROUP="docker"
+
+case "$1" in 
+    xeon)
+        PLATFORM=Xeon
+        ;;
+    vcac-a)
+        PLATFORM=VCAC-A
+        ;;
+    *)
+        PLATFORM=Xeon
+        ;;
+esac
```

Comment:
could skip first case if default is going to be Xeon

Reply:
done